tests/unit/helpers.js

```diff
@@ -1470,3 +1474,68 @@ export const createExperimentData = ({
 }) => {
   return { [id]: variantId };
 };
+
+const queryByClassName = (container, className) => {
+  return container.querySelectorAll(`.${className}`);
+};
+
+const getByClassName = (container, className) => {
+  const elements = queryByClassName(container, className);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for className ${className}`);
+  }
+  return elements[0];
+};
+
+const queryByTagName = (container, tagName) => {
+  return container.getElementsByTagName(tagName);
+};
+
+const getByTagName = (container, tagName) => {
+  const elements = queryByTagName(container, tagName);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for tag ${tagName}`);
+  }
+  return elements[0];
+};
+
+export const render = (ui, options = {}) => {
+  const internalOptions = {
+    history: addQueryParamsToHistory({ history: createMemoryHistory() }),
+    i18n: fakeI18n(),
+    store: dispatchClientMetadata().store,
+    rtlOptions: {},
```

Comment:
Why is this needed here?

Reply:
To be honest, I'm not sure. 😉 I think it is a remnant of when I was originally working on this stuff last year. I wanted to make it possible to override the options that eventually get passed to react-testing-library's `render` function, but I'm not sure if we will ever need to do that, and if there is a rare edge case, we could just do everything manually without the helper. I will look into this a bit more, but maybe we can just remove this.

Reply:
We can remove it, and I have done so.